README.md

```diff
@@ -12,9 +12,10 @@
       src="https://cdn.rawgit.com/webpack/media/e7485eb2/logo/icon.svg">
   </a>
   <h1>UglifyJS Webpack Plugin</h1>
-	<p>This plugin uses <a href="https://github.com/mishoo/UglifyJS2/tree/v2.x">UglifyJS v2</a> to minify your JavaScript.<p>
```

Comment:
Yep we need to specify it in `webpack-defaults`aswell then, but the question is still how we should do it :). Would a  e.g `.trim()` somewhere in webpack/webpack.js.org bee enough to avoid the parsing error (as `code` block) for now? If I'm not mistaken most READMEs use `<p>${Description}</p>` atm 😟, so this could be a lot of chore if unresolveable logic wise 😥